releasenotes/notes/only-add-log-handler-when-not-already-configured-60cb797f5381a6ef.yaml

```diff
@@ -0,0 +1,8 @@
+---
+fixes:
```

Comment:
If we put the stream handler behind a flag that you can use to toggle it off, is it a feat or a fix?

Reply:
I think it's open to interpretation. Personally I think a "fix" is appropriate here because this change is in response to a bug report.